app-emulation/libvirt/libvirt-9999.ebuild

```diff
@@ -276,6 +275,7 @@ src_configure() {
 
 		--localstatedir="${EPREFIX}/var"
 		-Drunstatedir="${EPREFIX}/run"
+		-Ddocdir="${EPREFIX}/usr/share/doc/${P}"
```

Comment:
We use `${PF}` for our docdirs. LGTM otherwise.

Reply:
Pushed fixed version.